iree/hal/cts/CMakeLists.txt

```diff
@@ -1,14 +1,27 @@
-################################################################################
-# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
-# iree/hal/cts/BUILD                                                           #
-#                                                                              #
-# Use iree_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary   #
-# CMake-only content.                                                          #
-#                                                                              #
-# To disable autogeneration for this file entirely, delete this header.        #
-################################################################################
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-iree_add_all_subdirs()
+# List of lists where each element is [test_name, TestClassName].
+#   * `test_name`s in underscore_case are used for target and file names.
+#   * `TestClassName` in TitleCase corresponds to subclasses of CtsTestBase.
+# An automated mapping and more implicit generation is certainly possible, but
+# this was simplest at the time of writing.
+set(IREE_ALL_CTS_TESTS
+  "allocator,AllocatorTest"
```

Comment:
can you make these consistent such that the mapping isn't needed? I feel like just making the C++ class names be lowercase and then dramatically simplifying all this would be fine - we're autogenerating files, after all, and can diverge from the style guide a bit :)

Reply:
Yeah, using lowercase class names would be simplest. I kept looking for ways to convert to TitleCase in CMake... and ugh...

Reply:
Switched to lowercase_test_names and removed this extra CMake hack.